portfolio/src/main/webapp/index.html

```diff
@@ -9,8 +9,39 @@
   <body>
     <div id="content">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+            <p> My name is Kevin Shao, and this is my portfolio! I live in Los Angeles, California, 
```

Comment:
In HTML, indents should be 2 spaces. You can use spaces instead of the tab keys if it's not already configured:

https://google.github.io/styleguide/htmlcssguide.html#Indentation